ArcadiaParties.API/Controllers/UsersController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using ArcadiaParties.CQRS.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Security.Principal;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArcadiaParties.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UsersController : ControllerBase
+    {        
+        private readonly IMediator _mediator;
+
+        public UsersController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [SwaggerOperation(
+             Summary = "Returns all users"
+        )]
+        [Authorize(Roles = "Administrator")]
+        [HttpGet]
+        [Route("GetUsers")]
+        public async Task<IActionResult> GetUsers(CancellationToken ct)
```

Comment:
Don't try to shorten every variable name. Your code will be read by other developers, not only by compiler, so code should be clearer and nicer.

Reply:
I've renamed this parameter